app/controllers/case_workers/claims_controller.rb

```diff
@@ -94,7 +95,7 @@ def claim_params
         disbursements
         vat_amount
       ]
-    ).merge(params.permit(:state_reason))
+    ).merge(params.permit(state_reason: []))
```

Comment:
can this b added to main permit? i.e.

```
params.require(:claim).permit(
  :state,
  :state_reason: []
  ...
```

Reply:
Good spot...
I just changed the data type and didn't look at the weird merging... :\

Reply:
reverted, but added reasons for the inconsistent approach